lib/utils/common-flags.ts

```diff
@@ -87,3 +90,38 @@ export const json: IBooleanFlag<boolean> = flags.boolean({
 	description: 'produce JSON output instead of tabular output',
 	default: false,
 });
+
+export const dataOutputFlags: flags.Input<DataOutputOptions> = {
+	fields: flags.string({
+		exclusive: ['extended'],
```

Comment:
Is there a `--extended` flag somewhere?

Reply:
Ah, that's what --verbose mapped to.  I'll remove that exclusive.

Reply:
Done